pkg/cmd/update.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright © 2020 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+	"regexp"
+
+	"github.com/DopplerHQ/cli/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+var updateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "update the Doppler CLI",
+	Args:  cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		if installedViaBrew() {
+			brewUpdate()
+		} else {
+			utils.HandleError(fmt.Errorf("this command is not yet implemented for your install method"))
+		}
```

Comment:
I think we can make this more modular for future package managers. Since I don't really know Go im going to write the suggestion in javascript.

``` js
switch(true) {
    case installedViaBrew(): return brewUpdate()
    case installedViaAPK(): return apkUpdate()
    default: throw new Error("this command is not yet implemented for your install method")
}
```

Reply:
The plan is to build it out as we support more. I'm just keeping it dead simple for now.

Reply:
okay sounds good.